docs/maps/asset-tracking-tutorial.asciidoc

```diff
@@ -270,47 +270,39 @@ PUT _ingest/pipeline/tri_met_tracks
 ----------------------------------
 
 [float]
-==== Step 2: Start {filebeat}
+==== Step 2: Configure {agent}
 
-. Replace the contents in your `filebeat.yml` file with the following:
-+
-[source,yaml]
-----------------------------------
-filebeat.inputs:
-# Fetch trimet bus data every minute.
-- type: httpjson
-  interval: 1m
-  request.url: "https://developer.trimet.org/ws/v2/vehicles?appID=<tri_met_app_id>"
-  response.split:
-    target: body.resultSet.vehicle
-  processors:
-    - decode_json_fields:
-        fields: ["message"]
-        target: "trimet"
+. From the {kib} main menu, click *Fleet*, then the *Agent policies* tab.
 
-  pipeline: "tri_met_tracks"
+. Click the name of the agent policy where you want to add the *Custom API* integration. The configuration changes you make only apply to the policy you select.
```

Comment:
should we expect users to have an agent policy? What if there are no agent policies?

Reply:
Then they will have to create one.

Reply:
If there are no policies it will prompt them to create one. Will update to reflect.